### PR TITLE
Dark Mode Issue #7

### PR DESCRIPTION
**Pull request by aaheli8**
### Description
Issue #7 
I have created a toggle switch that changes the theme from light mode to dark mode and vice versa.

### Checklist

- [x] I am making a proper pull request, not spam.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests
Issue #7 

## Add relevant screenshot or video (if any)
The moon button is the switch if we click on it, the theme changes to dark mode.
image

The Dark mode
image

(Add here)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -32,6 +32,24 @@
...
+
+}
+.dark-theme{
+	--primary-color: #171010;
+	--secondary-color : #fcf4f4;
+	--extra-color:#2B2B2B;
+}
+.bg{
+	background-color: var(--primary-color);
+}
```
Comment (wasimreja) [inline style.css:52]: Add background transition 
```suggestion
transition: background 500ms;
}

```

**Review by wasimreja — CHANGES_REQUESTED** [review]
change color appearance of the search bar and it would be better if you could make the dark mode toggle more minimalist like this https://letsbuildui.dev/articles/building-a-dark-mode-theme-toggle

**Comment by wasimreja** [comment]
@aaheli8 keep the dark mode toggle in the navbar and no need to design it as i previously said instead you can keep it simple like this https://wasimreja.me/

File: style.css (hunk 1 above)
Comment (aaheli8) [reply style.css:52]: Okay, I am making the necessary changes and keeping it in the navbar

**Comment by aaheli8** [comment]
I am working on it

**Pull request closed without merging**